src/invoices/InvoiceLineForm/InvoiceLineForm.js

```diff
@@ -70,14 +98,150 @@ class InvoiceLineForm extends Component {
           >
             <Row>
               <Col xs={12} md={8} mdOffset={2}>
-                <Field
-                  component={TextField}
-                  label={<FormattedMessage id="ui-invoice.invoiceLine.description" />}
-                  name="description"
-                  required
-                  type="text"
-                  validate={validateRequired}
-                />
+                <Row end="xs">
+                  <Col xs={12}>
+                    <ExpandAllButton accordionStatus={sections} onToggle={this.expandAll} />
+                  </Col>
+                </Row>
+                <AccordionSet accordionStatus={sections} onToggle={this.toggleSection}>
+                  <Accordion
+                    id={SECTIONS.invoiceLineInformation}
+                    label={<FormattedMessage id="ui-invoice.invoiceLineInformation" />}
+                  >
+                    {metadata && <ViewMetaData metadata={metadata} />}
+                    <Row>
+                      <Col data-test-col-invoice-line-description xs={3}>
```

Comment:
could be broken into dif lines here and below in this file

Reply:
And do you really need data-attribute for tests  instead of `name`

Reply:
I need this to reduce duplication attribute on Sonar, since there is no other legal and meaningful way to do it (at least what I or other folio dev know). And it's the same reason I keep it on the one line - to make these layout blocks different to code dup finder.

Reply:
Ok, got it. thank you